src/Stryker.CLI/Stryker.CLI/StrykerCLI.cs

```diff
@@ -51,10 +49,11 @@ public int Run(string[] args)
             {
                 // app started
                 PrintStrykerASCIIName();
-                PrintStrykerVersionInformationAsync();
 
-                var inputs = InputBuilder.Build(args, app, cmdConfigHandler);
+                InputBuilder.Build(inputs, args, app, cmdConfigHandler);
```

Comment:
It feels a bit weird that this is a static invocation that takes the inputs it created as params to it's own builder method. Perhaps we should make the InputBuilder non-static and save the inputs as a field.

Reply:
Agreed. But I went another way to solve it. The strykerinputs are now created in the strykerinputs (where else right?). And I splisted the InputBuilder into a configreader and a loggingInitializer. Better like this?

Reply:
@rouke-broersma if you agree I think this PR can be merged